src/index.ts

```diff
@@ -334,6 +340,51 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (nomodule) {
+      const nomoduleStart = Date.now();
```

Comment:
Minor point: I’m not crazy about `Date.now()` for performance profiling.

In exciting news, the [performance API](https://nodejs.org/api/perf_hooks.html#perf_hooks_class_performance) landed in Node 13, but I know not everyone is building snowpack on the new hotness. Still, that’d be cool if there would be a way to conditionally load that module if it existed.

Reply:
Ooooo I hadn't seen that. Awesome, can't wait to start using it once that gets out in the world more